### PR TITLE
add feature to delete selected records

### PR DESCRIPTION
**Pull request by RabiaSajjad**

### CODE REVIEW

**Comment by JVickery-TBS** [comment]
@RabiaSajjad as Recombinant does not actually have any datatable views stuff in it, it might be best to just put all of this into the Canada extension.

**Comment by JVickery-TBS** [comment]
Like we have the edit and create single records in the Canada extension.

File: ckanext/recombinant/views.py [hunk 1]
```diff
@@ -190,6 +190,72 @@ def record_fail(err):
         )
 
 
+@recombinant.route('/recombinant/delete-selected-records/<resource_id>', methods=['POST'])
```
Comment (JVickery-TBS) [inline ckanext/recombinant/views.py:193]: Need to add `GET` to allowed methods for CKAN 2.10

**Review by JVickery-TBS — CHANGES_REQUESTED** [review]

**Comment by RabiaSajjad** [comment]
Moved to Canada extension https://github.com/open-data/ckanext-canada/pull/1498

**Pull request closed without merging**